rollouts/pkg/packagediscovery/github_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build github
+
+package packagediscovery
+
+import (
+	"context"
+	"testing"
+
+	gitopsv1alpha1 "github.com/GoogleContainerTools/kpt/rollouts/api/v1alpha1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGitHubGetPackages_SingleRepo(t *testing.T) {
+	config := gitopsv1alpha1.PackagesConfig{
+		SourceType: gitopsv1alpha1.GitHub,
+		GitHub: gitopsv1alpha1.GitHubSource{
+			Selector: gitopsv1alpha1.GitHubSelector{
+				Org:       "droot",
```

Comment:
Should we create these test packages under `https://github.com/GoogleContainerTools/kpt/tree/main/package-examples` (or some other place under GoogleContainerTools)? I think using your username is ok short-term, but long term it would probably be best to have it somewhere more official and not tied to someone's individual account

Reply:
So two things:
1. Absolutely agree on keeping it in an official place. Mine is just a stop gap because of lack of a place.
2. We can't use `...kpt/tree/main/package-examples` for this because it complicates adding/deleting tests. One has to make changes in the test packages and commit it, then update the tests because tests will use whatever is available on `main`. `kpt-samples` could be another place but that is used for reference packages and using it for tests can make it really noisy.

But need to find a solution.

Reply:
Perhaps a `kpt-test` repo?